### ISSUE TITLE
Support for multiple languages

### ISSUE THREAD

**Issue by idontlikecreatingaccounts** [searxng/searxng#3010]
I know the "all languages" option exists. It's virtually useless because so many searches have results in a wide variety of languages. Just because I speak more than one languages doesn't mean I want a German news story and a Thai wikipedia article and a Portuguese message board thread about the thing I searched for.

The "auto-detect language" option exists, but it's very finnicky. I get "auto-detected" as en-US 75% of the time despite not even using English words in my query.

This issue was briefly touched upon at https://github.com/searxng/searxng/issues/2734, but I'm opening this ticket because the answer posited there is an obvious lie. It also doesn't provide any explanation for this weird statement. Not a priority? Maybe. Difficult? Perhaps. Not possible? Not true.

Google does it. If you wanted to, you could brute force a solution by performing two searches then combining the results. Or performing an "all languages" search, then filtering the results through the language detection feature that already exists. Clearly not impossible.

**TL:DR** Like an all languages search, but pick the languages.

**Comment by unixfox** [comment]
https://github.com/searxng/searxng/issues/2734#issuecomment-1712313444

That would be bad for response time and require to merge two engine request at the same time which is not supported.

> TL:DR Like an all languages search, but pick the languages.

That's not how it works, there is no search for all languages it just ask the engine for the results without specifying the language.
